OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -63,6 +63,15 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Arc in WAngles, two values indicate variable arc.")]
 		public readonly WAngle[] LaunchAngle = { WAngle.Zero };
 
+		[Desc("Up to how many times does this bullet bounce when touching ground without hitting a target?")]
+		public readonly int BounceCount = 0;
+
+		[Desc("Modify distance of each bounce by this percentage of previous distance.")]
+		public readonly int BounceRangeModifier = 60;
+
+		[Desc("If projectile touches an actor with one of these stances during or after the first bounce, trigger explosion.")]
+		public readonly Stance ValidTriggerStances = Stance.Enemy | Stance.Neutral;
```

Comment:
In this case, I agree that the projectile is what's needing the stance. However, I'm not certain what happens if the firer player dies during projectile path.